### PR TITLE
Feat: Added meeting duration to meeting model

### LINKED ISSUE #50: [Enhancement]: Add estimated meeting duration
### My idea

1. Add new field to `Meeting` model, that holds estimated meeting duration (probably string, so I could type `30-2137 min`)
2. Display it after description in `/gwardia`

### PR DESCRIPTION
**Pull request by tapczanek1321**
pls accept uwu ❤️

### CODE REVIEW

File: gwardia/models.py [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Meeting(models.Model):
     title = models.CharField(max_length=150)
     description = models.TextField()
     date = models.DateTimeField("Date of meeting")
+    duration = models.CharField(max_length=30, default="0oO")
```
Comment (Norbiros) [inline gwardia/models.py]: Why `0oO`?

Comment (Norbiros) [inline gwardia/models.py]: Remove default value.

File: gwardia/migrations/0005_meeting_duration.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+    dependencies = [
+        ("gwardia", "0004_rename_nextmeeting_meeting"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="meeting",
+            name="duration",
+            field=models.CharField(default="0oO"),
```
Comment (Norbiros) [inline gwardia/migrations/0005_meeting_duration.py]: And here :3

**Review by Norbiros — CHANGES_REQUESTED** [review]

**Review by Norbiros — DISMISSED** [review]

File: gwardia/migrations/0005_meeting_duration.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+    dependencies = [
+        ("gwardia", "0004_rename_nextmeeting_meeting"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="meeting",
+            name="duration",
+            field=models.CharField,
```
Comment (Sanielia) [inline gwardia/migrations/0005_meeting_duration.py]:
```suggestion
            field=models.CharField(
                default=datetime.timedelta(seconds=3600), max_length=30
            ),
            preserve_default=False,
```

**Review by Sanielia — CHANGES_REQUESTED** [review]
:3

**Review by Sanielia — DISMISSED** [review]
UwU

File: gwardia/templates/gwardia/list_events.html [hunk 4]
```diff
@@ -7,8 +7,10 @@ <h2 class="text-2xl text-center font-bold my-4">{{ title }}</h2>
                     <p class="text-orange-300 text-right w-full ">Archiwalne</p>
                 {% endif %}
                 <p class="w-full">
-                    <span class="break-words font-bold">{{ event.title }}</span>
                     <span class="float-right">{{ event.date }}</span>
+                    <br>
+                    <span class="break-words font-bold">{{ event.title }}</span>
+                    <span class="float-right">Czas trwania: {{ event.duration }} </span>
```
Comment (goteusz-maszyk) [inline gwardia/templates/gwardia/list_events.html]: _przewidywany czas trwania_

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

**Review by Norbiros — DISMISSED** [review]

File: gwardia/templates/gwardia/list_events.html (hunk 4 above)
Comment (tapczanek1321) [reply gwardia/templates/gwardia/list_events.html]: KYS

**Review by goteusz-maszyk — APPROVED** [review]

**Review by Sanielia — APPROVED** [review]

**Review by Norbiros — APPROVED** [review]

**Pull request merged**